packages/runtime/container-runtime/src/summary/summarizerTypes.ts

```diff
@@ -104,7 +104,8 @@ export interface ISummarizeOptions {
 	/**
 	 * True to ask the server what the latest summary is first; defaults to false
 	 *
-	 * @deprecated - Summarize will not refresh latest snapshot state anymore.
+	 * @deprecated - Summarize will not refresh latest snapshot state anymore. Instead it updates the cache and closes
```

Comment:
When's this getting removed outright?

Reply:
Not sure, The fact that we have retries we will still want a boolean. I would like to rename this, but that changes the API surface, and I'm happier just leaving this as is.

Reply:
k.  Taking a close look - After the new retry thing Navin made rolls out and becomes default I think we can probably remove it.